src/DocumentFormat.OpenXml/Packaging/OpenXmlPartContainer.cs

```diff
@@ -1777,11 +1777,8 @@ internal bool DeletePartCore(string id)
             var liveParts = new Dictionary<OpenXmlPart, bool>();
             var processedParts = new Dictionary<OpenXmlPart, bool>();
 
-            var child = GetPartById(id);
-
-            if (child == null)
+            if(!TryGetPartById(id, out var child) || child == null)
```

Comment:
child be null if it failed; any reason why you need to check it again?

Reply:
My bad, fixed now 👍